node/grpc/utils.go

```diff
@@ -55,16 +55,25 @@ func GetBlobMessages(in *pb.StoreChunksRequest, numWorkers int) ([]*core.BlobMes
 			}
 
 			bundles := make(map[core.QuorumID]core.Bundle, len(blob.GetBundles()))
-			for j, chunks := range blob.GetBundles() {
+			for j, bundle := range blob.GetBundles() {
 				quorumID := blob.GetHeader().GetQuorumHeaders()[j].GetQuorumId()
-				bundles[uint8(quorumID)] = make([]*encoding.Frame, len(chunks.GetChunks()))
-				for k, data := range chunks.GetChunks() {
-					chunk, err := new(encoding.Frame).Deserialize(data)
+				if len(bundle.GetBundle()) > 0 {
```

Comment:
can we have a get format function, it looks a bit adhoc

Reply:
There is no field to indicate the format, it's checking bundle field and then fallback to the old chunk encoding